FE/src/components/IssueTable/Issue/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as S from '@components/IssueTable/Issue/Issue.style';
+import IssueDescription from '@components/IssueTable/Issue/IssueDescription';
+import IssueTitle from '@components/IssueTable/Issue/IssueTitle';
+import CheckBox from '@components/common/CheckBox';
+import Image from '@components/common/Image';
+import { IMAGE_SIZE } from '@components/common/Image/constants';
+import { IssueListType } from '@type/issueType';
+
+const Issue = ({ title, labels, num, milestone, assignees, author, time }: IssueListType) => {
+  return (
+    <S.Wrapper>
+      <CheckBox checkBoxId='1' isChecked={true} toggleIsChecked={() => {}} />
+      <S.Container>
+        <IssueTitle {...{ title }} {...{ labels }} />
```

Comment:
명확하게 props 를 넘겨주는 건 어떠신가요???
```suggestion
        <IssueTitle title={title} labels={labels} />
```